network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/text_only_teaser_block.html

```diff
@@ -0,0 +1,24 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+	<div class="tw-row tw-py-4">
+		{% for block in self.cards %}
+          {% with card=block %}
+            <div class="tw-px-4 medium:tw-w-1/3 tw-mb-5">
+                <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
+                {% if card.heading_link %}<a href="{% pageurl card.heading_link %}">{% endif %}
+                    <h5 class="tw-h5-heading tw-mb-2">{{ card.heading }}</h5>
+                {% if card.heading_link %}</a>{% endif %}
+                {% if card.meta_data %}
+                <div class="tw-flex tw-flex-wrap">
+                    <span class="tw-h6-heading tw-text-gray-40 tw-py-1 tw-mb-1">{{ card.meta_data }}</span>
+                </div>
+                {% endif %}
```

Comment:
What do you think about indenting the content of the template tag here? I often find that more readable than only indenting got HTML tags.

Reply:
:+1: fixed in ebb22efa251f5f2490b2fd5835e8b2b2d62c18d9